requirements.txt

```diff
@@ -6,7 +6,7 @@ flake8
 # https://github.com/pallets/markupsafe/issues/284
 markupsafe==2.0.1
 # https://github.com/pandas-dev/pandas/pull/45749
-pandas==1.1.5
+pandas
```

Comment:
Can you give some context on why this changed? I'm assuming pandas 1.1.5 doesn't work with python3.10? Regardless we definitely need to still specify a minimum version of pandas (applies to setup.py too), but I don't know what that version is.

Reply:
So there's no pre-compiled wheel for pandas 1.1.5 on python 3.10, so when trying to build it from source the `Install python deps in venv` step would spin on forever (don't have a good illustration here on circleci since I did this in the SSH but this is an illustration of it running for at least 4 minutes):
https://app.circleci.com/pipelines/github/bluelabsio/records-mover/2234/workflows/362482c7-4ce4-40d1-8259-28839833798a/jobs/18590

Therefore, I decided to install this without a version number. I think you're right that it makes more sense to make this >= 1.1.5 as 1.1.5's minimum version is lower than the lowest currently supported python version:
https://pypi.org/project/pandas/1.1.5/#files
https://devguide.python.org/versions/